static/js/components/ShiftManagement.jsx

```diff
@@ -111,10 +446,10 @@ function CurrentShiftMenu() {
     // create list names of non admin members
     admins = currentShift.shift_users
       .filter((user) => user.admin)
-      .map((user) => `${user.first_name} ${user.last_name}`);
+      .map((user) => `${user.username}`);
```

Comment:
Issue here is that only the username is a required field, so users (for example in the demo data) that has no first and last name would show up as `null null`.

Reply:
Ok but this is a good chance to shame those who haven't given their full name, and we can modify the demo data if it is too annoying. The usernames are often simply unhelpful for identifying people when based on their emails.

Reply:
@mcoughlin should we do something in between: i display `first_name last_name (username)` for people that have first and last name, otherwise just `username` ? I agree that a username is not very helpful to identify people. Especially image that you are on shift when there is a gcn alert, and when writing the circular you just have the username of people ? Doesnt make any sense.

Reply:
I'll add my name to the demo data then :eyes:

Reply:
Since you and @guynir42 reviewed this i'll add both of your names to the demo data :crown:

Reply:
I love being famous.

Reply:
> @mcoughlin should we do something in between: i display `first_name last_name (username)` for people that have first and last name, otherwise just `username` ? I agree that a username is not very helpful to identify people. Especially image that you are on shift when there is a gcn alert, and when writing the circular you just have the username of people ? Doesnt make any sense.

I added this @mcoughlin as well as the other changes you asked for. Ready for review.